std_msgs/Header header
ObjectDetection[] detections
